tests/test_phase_psf.py

```diff
@@ -52,8 +53,51 @@ def test_aperture():
 
 
 @timer
+def test_atm_screen_size():
+    """Test for consistent AtmosphericScreen size and scale."""
+    screen_size = 10.0
+    screen_scale = 0.1
+    atm = galsim.AtmosphericScreen(screen_size=screen_size, screen_scale=screen_scale)
+    # AtmosphericScreen will preserve screen_scale, but will adjust screen_size as necessary to get
+    # a good FFT size.
+    assert atm.screen_scale == screen_scale
+    assert atm.screen_size != screen_size
```

Comment:
Rather than just the `!=` test, maybe also confirm that `screen_size < atm.screen_size < 1.5 * screen_size`, which I think should be true based on how `goodFFTSize` works.